src/main/java/com/loofah/graph/api/resources/SkillsResource.java

```diff
@@ -0,0 +1,30 @@
+package com.loofah.graph.api.resources;
+
+import com.loofah.graph.api.services.SkillsService;
+import graphql.ExecutionResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequestMapping("/skills")
+@RestController
+public class SkillsResource {
+
+    private SkillsService graphQLService;
+
+    @Autowired
+    public SkillsResource(SkillsService graphQLService) {
+        this.graphQLService = graphQLService;
+    }
+
+    @GetMapping
+    public ResponseEntity<Object> getSkills(@RequestParam String query) {
+        ExecutionResult executionResult = graphQLService.getGraphQL().execute(query);
```

Comment:
I think it might be neater/more encapsulated if the `SkillsService` had an `execute` method or similar, so we didn't have to reach through to the GraphQL object from the controller.

Reply:
Nice yep I've updated